lib/snippets.js

```diff
@@ -117,13 +120,13 @@ export const get${pluralize(
     return result;
   },
   updateRecord: (tableName) => `
-export const update${tableName} = async (id, recordUpdates) => { 
+export const update${tableName} = async (id, recordUpdates) => {
```

Comment:
hm maybe we should like....just run prettier or something in all outputted files..... but as long as it works i’m down!

Reply:
LOL this comment was meant for the earlier edits about formatting

Reply:
(when testing this I realized prettier was running on all my files anyway LOL so I had to turn it off but I guess consistency in the snippets or whatever?!)